spec/presenters/summary/sections/overview_spec.rb

```diff
@@ -42,36 +42,75 @@
       end
 
       it 'does not show this section' do
-        expect(subject.show?).to be(false)
+        expect(subject.show?).to be(true)
       end
     end
   end
 
   describe '#answers' do
     let(:answers) { subject.answers }
 
-    it 'has the correct rows' do
-      expect(answers.count).to eq(5)
+    context 'when the application is completed' do
+      it 'has the correct rows' do
+        expect(answers.count).to eq(6)
+
+        answer = answers[0]
```

Comment:
We have that pattern elsewhere that utilises a method to eliminate some of the duplication